src/main/kotlin/com/yapp/yapp/record/domain/Pace.kt

```
@@ -39,6 +39,11 @@ class Pace(
     constructor(distanceMeter: Double, durationMills: Long) :
         this(calculatePace(distanceMeter, durationMills))
 
+    fun toTts(): String {
+        val seconds = TimeProvider.millsToSecond(toMills())
+        return String.format("%d분 %d초", seconds / 60, seconds % 60)
```

detekt (1.23.8)

[warning] String.format("%d분 %d초", seconds / 60, seconds % 60) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)